Dockerfile

```diff
@@ -1,19 +1,20 @@
 # Use the official Docker Hub Ubuntu 14.04 base image
 FROM ubuntu:16.04
 
-# Update the base image
-RUN apt-get update
-RUN apt-get -y upgrade
-RUN apt-get -y dist-upgrade
-
 # Install Timesketch dependencies
-RUN apt-get -y install python-pip python-dev libffi-dev python-psycopg2
+RUN apt-get update \
+  && apt-get -y install python-pip \
+                        python-dev \
+                        libffi-dev \
+                        python-psycopg2 \
+                        software-properties-common
 
 # Install Plaso
-RUN apt-get -y install software-properties-common
-RUN add-apt-repository ppa:gift/stable
-RUN apt-get update
-RUN apt-get -y install python-plaso
+RUN add-apt-repository ppa:gift/stable \
+  && apt-get update \
```

Comment:
It's recommended to use [fewer layers](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/) in a Dockerfile to reduce the build time. The number of layers/keeping readability is subjective and depends on taste, but I tried to logically separate `RUN` parts. I can add comments or revert this if you wish.